### PR TITLE
getAll method for SDK

### PR DESCRIPTION
**Pull request by steve8708**

### CODE REVIEW

File: packages/core/src/builder.class.ts [hunk 1]
```diff
@@ -2033,6 +2033,10 @@ export class Builder {
     if (!this.apiKey) {
       throw new Error('Builder needs to be initialized with an API key!');
     }
-    return this.queueGetContent(modelName);
+    return this.queueGetContent(modelName, options);
+  }
+
+  getAll(modelName: string, options: GetContentOptions = {}) {
```
Comment (mandx) [inline packages/core/src/builder.class.ts:2039]: Does the user still needs to add a `.promise()` to this call to be used with `await`?

Comment (mandx) [reply packages/core/src/builder.class.ts:2039]: Also how about pagination?

Comment (steve8708) [reply packages/core/src/builder.class.ts:2043]: ah that's a good idea to .promise() by default for this

for pagination this supports the `offset` and `limit` params

Comment (mandx) [reply packages/core/src/builder.class.ts:2043]: Since this is public API, can I suggest to add return types to this method?

Comment (steve8708) [reply packages/core/src/builder.class.ts]: donzeo

**Pull request merged**